tutorial HowTo add an institution.md (#705)
nf-core/nf-co.re: markdown/usage/institution.md

## Adding your institution to the contributor's list

> Haven't you find your organization / group of contributors on the [community pages](https://nf-co.re/community)? Please add yourself! Just few lines of a YAML file needed.

Reviewer: ```suggestion
In case you couldn't find your organization / group of contributors on the [community pages](https://nf-co.re/community), please add yourself! Just a few lines of YAML and some icon / logos needed.
```
